docs/framework/data/adonet/ef/language-reference/math-canonical-functions.md

```diff
@@ -10,15 +10,15 @@ ms.assetid: 6f6cddc6-b561-4ebe-84b6-841ef5b4113b
   
 |Function|Description|  
 |--------------|-----------------|  
-|`Abs(` `value` `)`|Returns the absolute value of `value`.<br /><br /> **Arguments**<br /><br /> An `Int16`, `Int32`, `Int64`, `Byte`, `Single`, `Double`, and `Decimal`.<br /><br /> **Return Value**<br /><br /> The type of `value`.<br /><br /> **Example**<br /><br /> `Abs(-2)`|  
-|`Ceiling(` `value` `)`|Returns the smallest integer that is not less than `value`.<br /><br /> **Arguments**<br /><br /> A `Single`, `Double`, and `Decimal`.<br /><br /> **Return Value**<br /><br /> The type of `value`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#EDM_CEILING](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/cs/entitysql.cs#edm_ceiling)]
+|`Abs(value)`|Returns the absolute value of `value`.<br /><br /> **Arguments**<br /><br /> An `Int16`, `Int32`, `Int64`, `Byte`, `Single`, `Double`, and `Decimal`.<br /><br /> **Return Value**<br /><br /> The type of `value`.<br /><br /> **Example**<br /><br /> `Abs(-2)`|  
+|`Ceiling(value)`|Returns the smallest integer that is not less than `value`.<br /><br /> **Arguments**<br /><br /> A `Single`, `Double`, and `Decimal`.<br /><br /> **Return Value**<br /><br /> The type of `value`.<br /><br /> **Example**<br /><br /> [!code-csharp[DP EntityServices Concepts#EDM_CEILING](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts/cs/entitysql.cs#edm_ceiling)]
  [!code-sql[DP EntityServices Concepts#EDM_CEILING](../../../../../../samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts/tsql/entitysql.sql#edm_ceiling)]|  
```

Comment:
unrelated to your PR, but important to fix: This is intended to be in the second column of the previous row (for Ceiling(value)).  (See https://review.docs.microsoft.com/en-us/dotnet/framework/data/adonet/ef/language-reference/math-canonical-functions?branch=pr-en-us-6494).